capuccino/capuccino_py.py

```diff
@@ -0,0 +1,28 @@
+from math import sqrt, floor, pow
```

Comment:
If you have to enumerate a lot of functions, it is easier to use: import math

Reply:
Amended, so that it would be clear from which library the functions are coming from.